tests/operator/operator.go

```diff
@@ -2406,67 +2406,6 @@ spec:
 				vm = nil
 			}
 		})
-
-		It("[test_id:3153][QUARANTINE] Ensure infra can handle dynamically detecting DataVolume Support", decorators.Quarantine, func() {
```

Comment:
Isn't this still relevant?

Reply:
No, this test checks that we fail to create vm if cdi doesnt exist, we removed the webhook so it shouldn't fail.
More over this is a test that shouldn't have existed, its very time and resource consuming for something that a UT can cover quit simply.

Reply:
Right, it should not fail but we should test the new expected behavior. I am ok if we have a unit test.

Reply:
@xpivarc if I convinced you i'll appreciate lgtm :)